samples/python-samples/requirements.txt

```
@@ -0,0 +1,3 @@
+aws-cdk-lib==2.126.0
+constructs>=10.0.0,<11.0.0
+cdklabs.generative-ai-cdk-constructs==0.1.119
```

consider unless fixing to the specific version is necessary.

```suggestion
cdklabs.generative-ai-cdk-constructs>=0.1.119
```